keps/sig-cli/2590-kubectl-subresource/README.md

```diff
@@ -197,17 +201,39 @@ For custom resources:
 
 ### Test Plan
 
-- Unit tests, integration and e2e tests will be added.
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Unit tests
+
+- `k8s.io/kubernetes/pkg/printers/internalversion`: `2023-01-12` - 71.2
+- `k8s.io/kubernetes/vendor/k8s.io/cli-runtime/pkg/resource`: `2023-01-12` - 70.9
+- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/edit`: `2023-01-12` - 100
+- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/get`: `2023-01-12` - 80.7
+- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/patch`: `2023-01-12` - 56.3
+- `k8s.io/kubernetes/vendor/k8s.io/kubectl/pkg/cmd/replace`: `2023-01-12` - 63.6
+
+##### Integration tests
+
+- `kubectl get`: [link to test coverage](https://github.com/kubernetes/kubernetes/blob/4802d7bb62c2623be8e4f940f6b5c1fcddd6c744/test/cmd/get.sh#L178-L184)
+
+##### e2e tests
 
 ### Graduation Criteria
 
-#### Alpha -> Beta Graduation
+#### Alpha
 
-- [ ] Collect user feedback on adding support of `--subresource` for `apply`
+- Add the `--subresource` flag to get, patch, edit and replace commands.
 
-#### Beta -> GA Graduation
+#### Beta
+
+- Gather feedback from users.
+
+#### GA
+
+- e2e tests are added.
```

Comment:
These tests are required for GA, so you want them being added at beta stage, since that will be one of reqs for GA promotion.

Reply:
Updated.